### PR TITLE
Feature/master applications rate limit

### PR DESCRIPTION
**Pull request by bpsushi**
resolves:  #889

### CODE REVIEW

File: backend/core/test/applications/applications.e2e-spec.ts [hunk 1]
```diff
@@ -158,6 +159,11 @@ describe("Applications", () => {
         ListingsModule,
         ApplicationsModule,
         TypeOrmModule.forFeature([Application, HouseholdMember]),
+        ThrottlerModule.forRoot({
+          ttl: 60,
+          limit: 5,
+          ignoreUserAgents: [/^node-superagent.*$/],
```
Comment (pbn4) [inline backend/core/test/applications/applications.e2e-spec.ts:165]: Why ignore node-superagent?

File: backend/core/.env.template [hunk 2]
```diff
@@ -1,6 +1,7 @@
 PORT=3100
 NODE_ENV=development
 DATABASE_URL=postgres://bloom-dev:CHANGE-ME@localhost:5432/bloom
+REDIS_URL=redis://127.0.0.1:6379/0
```
Comment (pbn4) [inline backend/core/.env.template:4]: I just noticed that Heroku added two URLs to the container config for Redis and one of them is SSL, so I think the connection to redis will be over a public internet and we should use SSL in that case.

**Review by pbn4 — CHANGES_REQUESTED** [review]

File: backend/core/test/applications/applications.e2e-spec.ts (hunk 1 above)
Comment (bpsushi) [reply backend/core/test/applications/applications.e2e-spec.ts:165]: This agent is used by all tests. Without this all tests hits rate limit and fails. Note that the test for rate limits i added in this PR is forcing different agent.

File: backend/core/.env.template (hunk 2 above)
Comment (bpsushi) [reply backend/core/.env.template]: @pbn4 redis over TLS applied

**Pull request merged**